### PR TITLE
[EDX-107] Separate SSE API ref

### PR DESCRIPTION
**Pull request by tbedford**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

Move SSE API.

* [EDX-54](https://ably.atlassian.net/browse/EDX-54).
* [EDX-107](https://ably.atlassian.net/browse/EDX-107).

## Review

Review locally.

### CODE REVIEW

**Comment by tbedford** [comment]
Apologies, commit message on [a777eab](https://github.com/ably/docs/pull/1396/commits/a777eab620fa828f72dd9d0554e8f615f88070d4) was incorrect. I ended up moving both 1.2 and 1.1 in one commit. Note, SSE is only present for 1.2 and 1.1. Thanks.

File: content/sse/index.textile [hunk 1]
```diff
@@ -101,178 +98,6 @@ An important thing to note here is that the EventSource API tries to auto-reconn
...
-request
-  .get(url)
-  .on('response', function(response) {
-    console.log(response.statusCode) // stream started, 200
-  })
-  .on('data', function(data) {
-    console.log('Envelope:', data.toString())
-  });
-```
+View the "Server-Sent Events object":/api/sse for the associated Client Library SDK API Reference.
```
Comment (m-hulbert) [inline content/sse/index.textile:103]: I think we should drop "Client Library SDK" here and also not refer to it as an object.

File: content/sse/versions/v1.1/index.textile [hunk 2]
```diff
@@ -99,178 +96,6 @@ An important thing to note here is that the EventSource API tries to auto-reconn
...
-request
-  .get(url)
-  .on('response', function(response) {
-    console.log(response.statusCode) // stream started, 200
-  })
-  .on('data', function(data) {
-    console.log('Envelope:', data.toString())
-  });
-```
+View the "Server-Sent Events object":/api/sse for the associated Client Library SDK API Reference.
```
Comment (m-hulbert) [inline content/sse/versions/v1.1/index.textile:101]: Same comment as for 1.2.

**Review by m-hulbert — CHANGES_REQUESTED** [review]
One minor change and good to go!

File: content/sse/index.textile (hunk 1 above)
Comment (tbedford) [reply content/sse/index.textile]: Good call. Fixed.

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**